elixir/apps/web/lib/web/live/actors/show.ex

```diff
@@ -200,39 +122,119 @@ defmodule Web.Actors.Show do
           </:action>
         </.table>
       </:content>
+    </.section>
 
-      <:danger_zone>
-        <.action_button
+    <.section>
+      <:title>Danger Zone</:title>
```

Comment:
Let's have a `<danger_zone />` component which is just a simple wrapper around `<.section />` with pre-set title?

Reply:
Sure 👍

Reply:
Created a `<danger_zone>` component and updated all of the pages to use the new component.